pkgs/applications/audio/faust/faust2supercollider.nix

```diff
@@ -0,0 +1,26 @@
+{
+faust
+, supercollider
+, makeWrapper
+, stdenv
+}@args:
+
+
+stdenv.mkDerivation ((faust.faust2ApplBase (args // {baseName = "faust2supercollider";})) // {
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = [ faust supercollider];
+
+  postInstall = ''
+  '';
+
+  postFixup = ''
+    # export parts of the build environment
+    for script in "$out"/bin/*; do
+       wrapProgram "$script" \
+        --set SUPERCOLLIDER_HEADERS "${supercollider}/include/SuperCollider/" \
+         --prefix PATH : "$PATH"
+     done
+   '';
+})
```

Comment:
```suggestion
{ faust
, supercollider
, makeWrapper
, stdenv
}@args:

stdenv.mkDerivation (faust.faust2ApplBase (args // {
  baseName = "faust2supercollider";
})) // {
  nativeBuildInputs = [ makeWrapper ];

  propagatedBuildInputs = [ faust supercollider ];

  postInstall = ''
  '';

  postFixup = ''
    # export parts of the build environment
    for script in "$out"/bin/*; do
      wrapProgram "$script" \
        --set SUPERCOLLIDER_HEADERS "${supercollider}/include/SuperCollider/" \
        --prefix PATH : "$PATH"
    done
  '';
}
```
meta is missing

Reply:
what is meta?

Reply:
removing the brackets passes the main attribute set not to mkDerivation

Reply:
> meta is missing

I thought it is sufficient to use the meta tags from [faust](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/faust/faust2.nix) like the other files in this folder.